es-svc.yaml

```diff
@@ -10,7 +10,9 @@ spec:
   selector:
     component: elasticsearch
     role: client
+  clusterIP: 10.0.0.110
```

Comment:
You can just use the DNS entry `elasticsearch` or whatever name you gave to the service exposing client nodes HTTP port (9200), so it would become `http://elasticsearch:9200`.